tests/e2e/test_e2e.py

```diff
@@ -32,3 +32,41 @@ def test_print_config_token(helper: Helper) -> None:
     captured = helper.run_cli(["config", "show-token"])
     assert not captured.err
     assert captured.out  # some secure information was printed
+
+
+@pytest.mark.e2e
+def test_root_trace_hide_token_default_true(helper: Helper) -> None:
```

Comment:
Please drop e2e tests.
CLI tests are enough, the time cost of e2e is too high; let's no pessimize the test execution time if not necessary.

Reply:
Should we replace these e2e with a unit test on `def cli()` or just believe it works as intended?

Reply:
I don't care

Reply:
I have dropped two e2e tests that check that options `--hide-token` and `--no-hide-token` require the option `--trace` as this case is minor in terms of importance, and two other e2e tests that check this functionality if the option  `--hide-token` or `--no-hide-token` was specified explicitly. 

The only test I want to keep is `test_root_trace_hide_token_default_true` because it checks the use-case I worry about: when the user runs `neuro --trace <command>` and their token is not revealed.